mpv-plugin/open-in-mpv/open-in-mpv-2.4.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module desktop xdg-utils
+
+DESCRIPTION="Simple host script (in Go) for simple web extension to open videos in mpv"
+HOMEPAGE="https://github.com/Baldomo/open-in-mpv https://addons.mozilla.org/en-US/firefox/addon/iina-open-in-mpv"
+SRC_URI="https://github.com/Baldomo/open-in-mpv/releases/download/v${PV}/open-in-mpv_${PV}.tar.gz"
+S="${WORKDIR}/${PN}_${PV}"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	media-video/mpv
```

Comment:
@vitaly-zdanevich you can also move this package to media-video category, since there is no mpv-plugin category.

Reply:
> there is no mpv-plugin category

https://github.com/gentoo/guru/tree/master/mpv-plugin

Reply:
> > there is no mpv-plugin category
> 
> https://github.com/gentoo/guru/tree/master/mpv-plugin

Oh, I judged based on standard categories from main gentoo repository. I think it's fine to keep it like this if any maintainers agree. All cool.